webapp/src/filelist.tsx

```diff
@@ -362,11 +362,13 @@ export class FileList extends data.Component<ISettingsProps, FileListState> {
                 return Promise.resolve()
             }
             let fileText = "";
-            if (fileName == customFile) {
+            const tsFiles = pkgCfg.files.filter(f => f.endsWith(".ts"));
+            // if the length is 1, the only ts file is main.ts, thus
+            // we are adding the first custom file to the project
+            if (tsFiles.length === 1) {
                 fileText = customFileHeader(pxt.appTarget.appTheme.homeUrl) + customFileText;
             } else if (comment) {
-                fileText = `${comment} ${commentText}
```

Comment:
nit: Was this newline here for stylistic (empty new line at the end of the file) reasons? If so, should we keep it?

Reply:
Ah, yeah, it is. Good call, I'll leave it in so we have that extra new line in the file.